<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="modInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QScrollArea" name="leftScroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>427</width>
           <height>500</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTextEdit" name="systemInput">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>12</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Система уравнений</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="calculateBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollArea" name="rightScroll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>12</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>426</width>
           <height>500</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="outputLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Результаты</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuField">
    <property name="title">
     <string>Выбор поля</string>
    </property>
    <addaction name="actionFieldR"/>
    <addaction name="actionFieldC"/>
    <addaction name="actionFieldGF"/>
    <addaction name="actionFieldZn"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Алгоритм</string>
    </property>
    <addaction name="actionBuch"/>
    <addaction name="actionBuchB"/>
    <addaction name="actionF5"/>
    <addaction name="actionF5C"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Упорядочение</string>
    </property>
    <addaction name="actionLEX"/>
    <addaction name="actionINVLEX"/>
    <addaction name="actionGRLEX"/>
    <addaction name="actionGREVLEX"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuField"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="actionFieldR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поле рациональных чисел</string>
   </property>
  </action>
  <action name="actionFieldC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поле комплексных чисел</string>
   </property>
  </action>
  <action name="actionFieldGF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поле Галуа характеристики 2</string>
   </property>
  </action>
  <action name="actionFieldZn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поле вычетов по модулю простого числа</string>
   </property>
  </action>
  <action name="actionBuch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Алг. Бухбергера</string>
   </property>
  </action>
  <action name="actionBuchB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Алг. Бухбергера улучшенный</string>
   </property>
  </action>
  <action name="actionF5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Алг. F5</string>
   </property>
  </action>
  <action name="actionF5C">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Алг. F5C</string>
   </property>
  </action>
  <action name="actionLEX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LEX</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionINVLEX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>INVLEX</string>
   </property>
  </action>
  <action name="actionGRLEX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GRLEX</string>
   </property>
  </action>
  <action name="actionGREVLEX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GREVLEX</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
